src/test/ui/moves/move-out-of-tuple-field.stderr

```diff
@@ -8,15 +8,15 @@ LL |     let z = x.0;
    |
    = note: move occurs because `x.0` has type `Box<i32>`, which does not implement the `Copy` trait
 
-error[E0382]: use of moved value: `x.0`
+error[E0382]: use of moved value: `x`
```

Comment:
Hm, can you investigate why this is still a regression?

Reply:
sure